### PR TITLE
Bug 1340321 - Add a toggle for a left-hand keyboard shortcuts mode

### PR DESCRIPTION
**Pull request by KWierso**
This change is [Reviewable](https://reviewable.io/reviews/mozilla/treeherder/2183)

### CODE REVIEW

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by KWierso** [comment]
This is up and running on my gh-pages branch: https://kwierso.github.io/treeherder/ui/#/jobs?repo=mozilla-inbound&gamermode

This PR so far toggles some keybindings when "&gamermode" is added to the URL:
Select next/previous job is now 'w'/'s'.
Select next/previous unclassified failure is now 'a'/'d'.
Open logviewer for selected job is now 'q'.
Toggle unclassified jobs is now 'e'.
Toggle in progress jobs is now 'v'.

I've added ctrl-space as a shortcut to save the classification in addition to ctrl-enter.
Since 'a' and 's' have been repurposed above, the autoclassify shortcuts for "ignore unclassified" and "save all autoclassifications" have been moved to 'z' and 'x', respectively. (But only in this new lefty/gamer mode.

With these keybindings, the only shortcuts I'm aware of that can't be accessed from just the left side of the keyboard plus the mouse are the ctrl-shift-u shortcut to clear (a bit ironically) the pinboard and the '?' shortcut to pop up the list of shortcuts.


I think there's probably a better way to do this than with a URL parameter (especially since it isn't currently in the list of parameters that gets saved between certain page loads, and there's no in-UI way to toggle it at the moment. Maybe I can add the toggle to the shortcut popup list itself, and it could be saved into localstorage or something?

File: ui/js/controllers/main.js [hunk 1]
```diff
@@ -478,16 +501,29 @@ treeherderApp.controller('MainCtrl', [
...
                     $scope.$evalAsync($rootScope.$emit(thEvents.saveAllAutoclassifications));
                 }
             }],
 
             // Shortcut: delete classification and related bugs
-            ['a', function() {
-                if (thTabs.selectedTab === "autoClassification") {
-                    $scope.$evalAsync($rootScope.$emit(thEvents.ignoreOthersAutoclassifications));
+            [['a', 'z'], function(ev) {
+                if (($scope.inGamerMode && ev.key === 'z') || $scope.inGamerMode === false && ev.key === 'a') {
```
Comment (camd) [inline ui/js/controllers/main.js]: easier to read if you put ``!$scope.inGamerMode && ev.key === 'a'``

**Comment by KWierso** [comment]
I don't see myself finishing this.

**Pull request closed without merging**